src/stepfunctions/steps/sagemaker.py

```diff
@@ -477,9 +478,15 @@ class ProcessingStep(Task):
 
     """
     Creates a Task State to execute a SageMaker Processing Job.
+
+    The following properties can be passed down as kwargs to the sagemaker.processing.Processor to be used dynamically
+    in the processing job (compatible with Placeholders): role, image_uri, instance_count, instance_type,
+    volume_size_in_gb, volume_kms_key, output_kms_key
     """
 
-    def __init__(self, state_id, processor, job_name, inputs=None, outputs=None, experiment_config=None, container_arguments=None, container_entrypoint=None, kms_key_id=None, wait_for_completion=True, tags=None, **kwargs):
+    def __init__(self, state_id, processor, job_name, inputs=None, outputs=None, experiment_config=None,
+                 container_arguments=None, container_entrypoint=None, kms_key_id=None, wait_for_completion=True,
+                 tags=None, **kwargs):
         """
         Args:
```

Comment:
Now that we accept `parameters`, document it here and describe what it is used for and how it relates to the rest of the args

Reply:
Updated the `parameters` doc [here](https://github.com/aws/aws-step-functions-data-science-sdk-python/pull/155/files#diff-bbe5592768d985044b425a113b47ee71849979921c0109dfa399e57e88f3bddbR506)